config-overrides.js

```diff
@@ -14,17 +14,6 @@ module.exports = {
         });
 
         if (env === 'production') {
-            oneOfRule.oneOf.splice(0, 0, {
-                test: /\.svg$/,
-                include: path.resolve(srcRoot, 'assets/illustrations'),
-                loader: 'file-loader',
-                options: {
-                    // default is 'static/media/...', but the embedded version static is served from 'resources/media/...',
-                    // and the 'resources' substring is handled by the publicPath below
-                    name: 'media/[name].[hash:8].[ext]',
-                },
-            });
-            config.output.publicPath = 'resources/';
```

Comment:
An example of more correct paths.

Imaging we have two pages:
- `/monitoring/tenants`
- `/monitoring/tablets/<tablet_id>`

imagine a file `script.js` loaded on both pages 

in the old version it would load twice, from these urls:
- `/monitoring/resources/js/script.js`
- `/monitoring/tablets/resources/js/script.js`

in the new version it would load once from this url:
- `/monitoring/static/js/script.js`

Reply:
Correction: discussions revealed that relative paths are crucial for the embedded version.
It can be hosted on a hub, with a proxy backend being at the start of URL, e.g. `/ydb-vla02-dev.yandex.net/monitoring/<...>`.
In this case backend would still receive `filename` as a string starting with `monitoring`, but if in UI paths were absolute, then this request would go to the hub backend, and not to the proxy backend, and will result in 404.